ethereum-network-partition-test/main_test.go

```diff
@@ -194,21 +232,17 @@ func partitionNetwork(t *testing.T, enclaveCtx *enclaves.EnclaveContext) {
 		partitionedNetworkServices[secondPartition][renderServiceId(clNodeValidatorIdTemplate, nodeIdForSecondPartition)] = true
 	}
 
-	partitionedNetworkConnections := map[enclaves.PartitionID]map[enclaves.PartitionID]enclaves.PartitionConnection{
+	partitionedNetworkConnections := map[enclaves.PartitionID]map[enclaves.PartitionID]float64{
 		firstPartition: {
-			secondPartition: blockedPartitionConnection,
+			secondPartition: blockedPartitionPacketLossPercentage,
 		},
 	}
 
-	err := enclaveCtx.RepartitionNetwork(
-		partitionedNetworkServices,
-		partitionedNetworkConnections,
-		blockedPartitionConnection,
-	)
+	err := repartitionNetworkUsingStarlark(ctx, enclaveCtx, partitionedNetworkServices, partitionedNetworkConnections)
 	require.NoError(t, err, "An error occurred repartitioning the network")
 }
 
-func healNetwork(t *testing.T, enclaveCtx *enclaves.EnclaveContext) {
+func healNetwork(t *testing.T, ctx context.Context, enclaveCtx *enclaves.EnclaveContext) {
```

Comment:
Yeah if you do the above suggestion this method becomes just be a single call to `set_connection((firstPartition, secondPartition), ALLOWED)` and that's it